src/nrepl/misc.clje

```diff
@@ -0,0 +1,63 @@
+(ns nrepl.misc
+  "Misc utilities used in nREPL's implementation (potentially also
+  useful for anyone extending it)."
+  {:author "Chas Emerick"}
+  (:refer-clojure :exclude [requiring-resolve]))
+
+(defn log
+  [ex & msgs]
+  (let [ex (when (instance? clojerl.ExceptionInfo ex) ex)
```

Comment:
I checked what the original code was doing. Since it's checking for `Throwable` the closer we can do in Clojerl is to check if  `(satisfies? clojerl.IError ex)`.

Reply:
OK, so `clojerl.IError` is the root type wrapper of everything that is thrown in Erlang?

Reply:
`clojerl.IError` is a protocol defined in Clojerl for the custom errors. In Erlang any value can be thrown as an exception.